download.go

```diff
@@ -357,7 +359,7 @@ func downloadFile(f *gdrive.File, localPath string, progressBar *pb.ProgressBar)
 
 	verbose.Printf("Downloaded and wrote %d bytes to %s", f.FileSize, localPath)
 
-	return os.Chtimes(localPath, normalizeModTime(f.ModTime), normalizeModTime(f.ModTime))
+	return err
```

Comment:
`err` is reassigned inside the defered func created on line 335. However, the logic is very convoluted (and incorrect if some other error occurred) so I already closed this PR.